presto-main/src/main/java/com/facebook/presto/spiller/SpillerStats.java

```diff
@@ -11,21 +11,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package com.facebook.presto.spiller;
 
 import org.weakref.jmx.Managed;
 
 import java.util.concurrent.atomic.AtomicLong;
 
-public abstract class SpillerFactoryWithStats
-        implements SpillerFactory
+public class SpillerStats
 {
     protected final AtomicLong totalSpilledBytes = new AtomicLong();
 
-    @Override
     @Managed
     public long getTotalSpilledBytes()
```

Comment:
It would probably make sense for the other stats that I added to be here in SpillerStats, too.